app/views/admin/editions/_speed_tagging.html.erb

```diff
@@ -1,66 +0,0 @@
-<div class="speed-tag">
-  <% unless @edition.valid_as_draft? %>
-    <div class="alert alert-danger">
-      <strong>This edition cannot be saved because:</strong>
-
-      <ul>
-        <% @edition.errors_as_draft.full_messages.each do |message| %>
-          <li><%= message %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-  <% unless @edition.document.document_sources.empty? %>
```

Comment:
I have a _sneaky suspicion_ that we'll also be able to get rid of Document Sources. But I think this might need some more digging, so perhaps one to tackle later. (I'd like to check with Kevin first!)

Reply:
Definitely worth investigating!